app/serializers/component_serializer/hint_component_serializer.rb

```diff
@@ -4,7 +4,10 @@ class HintComponentSerializer < ComponentSerializer::BaseComponentSerializer
     #
     # @param [String] content a translation block that is evaluated in the front-end
     # @param [Array<Hash>] display_data all the css classes for the component
-    def initialize(content: nil, display_data: [])
+    #
+    # @example Initialising a hint partial
```

Comment:
If you're providing a single example, you don't need to provide a name too - I'd compare the generated HTML with and without a name

Reply:
It may not be 100% necessary but I think it makes the Docs more readable.

Reply:
I'm happy with that :)